lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/LionWebModelConverter.kt

```diff
@@ -3,40 +3,20 @@ package com.strumenta.kolasu.lionweb
 import com.strumenta.kolasu.ids.IDGenerationException
 import com.strumenta.kolasu.ids.NodeIdProvider
 import com.strumenta.kolasu.language.KolasuLanguage
-import com.strumenta.kolasu.model.CompositeDestination
-import com.strumenta.kolasu.model.Multiplicity
-import com.strumenta.kolasu.model.Point
-import com.strumenta.kolasu.model.Position
-import com.strumenta.kolasu.model.PossiblyNamed
-import com.strumenta.kolasu.model.ReferenceByName
-import com.strumenta.kolasu.model.allFeatures
-import com.strumenta.kolasu.model.asContainment
-import com.strumenta.kolasu.model.assignParents
-import com.strumenta.kolasu.model.isAttribute
-import com.strumenta.kolasu.model.isContainment
-import com.strumenta.kolasu.model.isReference
-import com.strumenta.kolasu.model.nodeOriginalProperties
+import com.strumenta.kolasu.model.*
```

Comment:
I thought ktlint would not like that. I configured my IDEA to avoid grouping imports in this way.
It is strange that the build is passing

Reply:
I thought I had configured IDEA like that too... however I recently updated it and can't find the option anymore (now apparently Kotlin is built into the IDE and it's no longer a separate plugin)